docs/monoDrive_home/Simulator-Commands.md

```diff
@@ -0,0 +1,365 @@
+			{ 
+				"R":(float), - the red component of the color
+				"G": (float), - the green component of the color
+				"B": (float), - the blue component of the color
+				"A": (float) - the alpha value (transparency) of the color
+			},
+			"SunIndirectLightIntensity": (float) - the intensity of the indirect sunlight,
+			"CloudOpacity": (float) - the cloud opacity,
+			"HorizontFalloff": (float) - the horizontal fall-off value,
+			"ZenithColor": (FLinearColor) - the zenith color,
+			"HorizonColor": (FLinearColor) - the horizon color,
+			"CloudColor": (FLinearColor) - the cloud color,
+			"OverallSkyColor": (FLinearColor) - the overall sky color,
+			"SkyLightIntensity": (float) - the intensity of the sky light,
+			"SkyLightColor": (FLinearColor) - the sky light color,
+			"bPrecipitation": (boolean) - whether to add precipitation,
+			"PrecipitationType": (string) - precipitation type (currently only "Rain" is supported),
+			"PrecipitationAmount": (float) - the amount of precipitation to add. The higher the number the more dense the precipitation,
+			"PrecipitationAccumulation": (float) - the amount of precipitation that accumulates on the ground,
+			"bWind": (boolean) - whether wind is present (affects precipitation),
+			"WindIntensity": (float) - the intensity of the wind,
+			"WindAngle": (float) - the angle of the wind, in degrees,
+			"bOverrideCameraPostProcessParameters": (boolean) - whether to override the camera post processing with the parameters below,
+			"CameraPostProcessParameters": 
+			{
+				"AutoExposureMethod": (string) - the luminance computation method (e.g. "Histogram"),
+				"AutoExposureMinBrightness": (float) - auto-exposure minimum adaptation,
+				"AutoExposureMaxBrightness": (float) - auto-exposure maximum adaptation,
+				"AutoExposureBias": (float) - logarithmic adjustment of the exposure
+			}
+		},
+		...  - additional profiles
+	]
+}
+```
+
+**response**: (string)
+
+`DynamicWeather is not available` if the weather actor is missing from the map
+`Weather configuration failed` if the weather profiles are invalid or mal-formed
+`Weather profile is invalid or missing` if the profile ID is not found
+`Weather configured to {profile_id}` if the weather was properly configured
+
+
+## Sensor Configuration
+Configures a sensor or set of sensors to use with the ego vehicle in the simulation run. The command can take a [single sensor configuration](Common.md), or an array of sensor configurations.
+
+**command ID**:  "REPLAY_ConfigureSensorsCommand_ID"
+
+**command data**:
+
+[sensor_config](Common.md)
+
+```json
+[
+	Sensor_config, ...
+]
+```
+
+**response**: (string)
+
+`complete` if the sensor was properly configured
+
+
+## Sensor Reconfiguration
+Reconfigures a previously configured sensor or set of sensors to use with the ego vehicle in the simulation run. The command takes a [single sensor configuration](Common.md).
```

Comment:
Can you state how to refer to a specific sensor here? If I had 5 cameras on a vehicle, how do I know that I'm reconfiguring the 3rd camera when I want to?